src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -100,13 +108,34 @@ protected override void OnPersist(ApplicationEngine engine)
             storage.Value = GetValidators(engine.Snapshot).ToByteArray();
         }
 
+        [ContractMethod(0_05000000, CallFlags.AllowModifyStates)]
+        private bool SetGasPerBlock(ApplicationEngine engine, BigInteger gasPerBlock)
+        {
+            if (gasPerBlock < 0 || gasPerBlock > 10 * GAS.Factor) return false;
+            if (!CheckCommittee(engine)) return false;
+            StorageKey key = CreateStorageKey(Prefix_GasPerBlock).AddBigEndian(uint.MaxValue - engine.Snapshot.PersistingBlock.Index - 1);
```

Comment:
Maybe the key should be the index of the block. I know the performance of `next()` is higher than `prev()`, but it is not worth such optimization here.

Reply:
No, and it's not very convenient to use the `DataCache.Seek` method.

If we want to seek(index = 10) in db, which contains 0, 8, 15.  It'll set position at 15.

Reply:
What about we save all values in a single entry? I think it won't be modified often.

Reply:
It's ok.

Reply:
It's good, I'll change it.

Reply:
Adjusted, but It doesn't look as good as before.